packages/core/src/app/shipping/ConsignmentListItem.tsx

```diff
@@ -42,15 +42,17 @@ const ConsignmentListItem: FunctionComponent<ConsignmentListItemProps> = ({
     return (
         <div className='consignment-container'>
             <div className='consignment-header'>
-                <h3><TranslatedString data={{ consignmentNumber }} id="shipping.multishipping_consignment_index_heading" /></h3>
-                    <a
-                        className="delete-consignment"
-                        data-test="delete-consignment-button"
-                        href="#"
-                        onClick={preventDefault(handleClose)}
-                    >
-                        <IconClose size={IconSize.Small}/>
-                    </a>
+                <h3>
+                    <TranslatedString data={{ consignmentNumber }} id="shipping.multishipping_consignment_index_heading" />
+                </h3>
+                <a
+                    className="delete-consignment"
+                    data-test="delete-consignment-button"
+                    href="#"
+                    onClick={preventDefault(handleClose)}
+                >
```

Comment:
Ident missing.

Reply:
`<h3>` and `<a>` tags are siblings here, so should be at same ident.

Reply:
Same...🤦🏻‍♂️